spec/features/can_registred_spec.rb

```diff
@@ -25,4 +25,40 @@
     click_on 'Регистрация'
     expect(page).to have_content 'Email has already been taken'
   end
+
+  scenario 'the user has entered invalid email' do
+    visit root_path
+    click_on 'Register'
+    fill_in 'user[email]', with: '111'
+    fill_in 'user[password]', with: 12345
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.